inheritance/src/main/java/inheritance/Review.java

```diff
@@ -6,7 +6,9 @@
     private String author;
     private double stars;
 
-    public Review(String body, String author, double stars) {
+
+
+    public Review(String name, String price, double stars) {
         super(name, price, stars);
     }
```

Comment:
your review does not need to extend restaurant. Your toString method is failing because you pass name, price, and stars through super to the Restaurant class. Review can just take in author, message and stars and contain those instance variables